src/pages/example1.md

```diff
@@ -7,96 +7,83 @@ layout: "@layouts/MarkdownPage.astro"
 <h3>Basic pipeline</h3>
 
 <p class="text-muted" >
-    This example shows how to write a pipeline with two simple Bash processes, so that the results produced by the first process are consumed by the second process.
+    This example shows a simple Nextflow pipeline consisting of two Bash processes.
 </p>
 
 ```groovy
 #!/usr/bin/env nextflow
 
-params.in = "$baseDir/data/sample.fa"
+/*
+ * Pipeline parameters
+ */
+
+// Primary input
+params.greeting = "Hello World!"
 
 /*
- * Split a fasta file into multiple files
+ * Redirect a string to a text file
  */
-process splitSequences {
+process sayHello {
 
     input:
-    path 'input.fa'
+    val x
 
     output:
-    path 'seq_*'
+    path 'output.txt'
 
+    script:
     """
-    awk '/^>/{f="seq_"++d} {print > f}' < input.fa
+    echo '$x' > output.txt
     """
 }
 
 /*
- * Reverse the sequences
+ * Convert lowercase letters to uppercase letters
  */
-process reverse {
+process convertToUpper {
 
     input:
-    path x
+    path y
 
     output:
     stdout
 
+    script:
     """
-    cat $x | rev
+    cat $y | tr '[a-z]' '[A-Z]'
     """
 }
 
 /*
- * Define the workflow
+ * Workflow definition
  */
 workflow {
-    splitSequences(params.in) \
-      | reverse \
-      | view
-}
-```
-
-</div>
-
-### Synopsis
-
-- **Line 1** The script starts with a shebang declaration. This allows you to launch your pipeline just like any other Bash script.
 
-- **Line 3**: Declares a pipeline parameter named `params.in` that is initialized with the value `$HOME/sample.fa`. This value can be overridden when launching the pipeline, by simply adding the option `--in <value>` to the script command line.
+    // Redirects a string to a text file
+    sayHello(params.greeting)
```

Comment:
Relying on Nextflow to auto-cast the String `params.greeting` into a file object is dangerous because it will break if the user sets `--greeting to/a/relative/path` (Issue [here](https://github.com/nextflow-io/nextflow/issues/5289)). Rather than let people discover this on their own, we should either use `Channel.fromPath` or wrap the String in the `file` function:

```suggestion
    greetingsFile = file(params.greeting)
    sayHello(greetingsFile)
```

Reply:
Another option is 

```
Channel.fromPath(params.greeting) | sayHello
```

Reply:
`sayHello` input is string, not a file. This might need to be added to a different example.

Reply:
Oh, right! Sorry! Fixing...